components/user/UserTabs.js

```diff
@@ -102,7 +102,7 @@ export default function UserTabs({ tabs, setTabs, setUserData, userData }) {
       </div>
       <div className="hidden sm:block">
         <div className="border-b border-gray-200">
-          <nav className="-mb-px flex" aria-label="Tabs">
+          <nav className="mb-4  flex" aria-label="Tabs">
```

Comment:
This looked better before, as the bottom border line is part of the tabs

BEFORE

<img width="758" alt="Screenshot 2023-02-07 at 14 24 26" src="https://user-images.githubusercontent.com/624760/217176615-6c6e3f6f-3fc8-4d3a-bb97-0f1263869169.png">


AFTER

<img width="731" alt="Screenshot 2023-02-07 at 14 24 31" src="https://user-images.githubusercontent.com/624760/217176667-8d174f9d-c851-4c80-aa67-950d6b4199df.png">

I suggest moving the content down, but not changing the Tailwind tabs

Reply:
okay will do it and raise a PR right away